sql/ashamane/world/2018_09_08_00_world_spell_areatrigger.sql

```diff
@@ -0,0 +1,3 @@
+/*Areatrigger*/
+DELETE FROM `spell_areatrigger` WHERE (`SpellMiscId`) IN ('9769');
+INSERT INTO `spell_areatrigger` (`SpellMiscId`, `AreaTriggerId`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `VerifiedBuild`) VALUES ('9769', '3769', '0', '0', '0', '0', '0', '0', '20000', '35666');
```

Comment:
Is this sniffed data ? If yes you should set VerifiedBuild to a valid value, if not set it to 0 (same for areatrigger_template)